build.sbt

```diff
@@ -79,16 +85,59 @@ libraryDependencies ++= Seq(
   "com.lihaoyi" %% "fastparse" % "1.0.0",
   "org.scalaj" %% "scalaj-http" % "2.4.2",
   // Google Dataproc's spark-bigquery-connector allows interacting with BigQuery tables on Dataproc
-   "com.google.cloud.spark" %% "spark-bigquery-with-dependencies" % "0.25.0",
+  "com.google.cloud.spark" %% "spark-bigquery-with-dependencies" % "0.25.0",
   // For reading Neo4j database files.
   "org.neo4j" % "neo4j-community" % "3.5.2",
+  "com.fasterxml.jackson.dataformat" % "jackson-dataformat-yaml" % "2.10.0",
+)
+
+excludeDependencies ++= Seq(
+  "commons-logging" % "commons-logging",
+  "javax.activation" % "activation",
+  "com.typesafe.akka" % "akka-protobuf-v3_2.12",
+)
+
+assemblyShadeRules in assembly := Seq(
+  ShadeRule.rename("com.typesafe.config.**" -> "lynxkite_shaded.com.typesafe.config.@1").inAll,
+  ShadeRule.rename("com.google.inject.**" -> "lynxkite_shaded.com.google.inject.@1").inAll,
+  ShadeRule.rename("com.google.common.**" -> "lynxkite_shaded.com.google.common.@1").inAll,
+  ShadeRule.rename("com.google.protobuf.**" -> "lynxkite_shaded.com.google.protobuf.@1").inAll,
 )
```

Comment:
This shading is motivated by running on Databricks where we find conflicting versions. It's probably important in other environments as well.